src/libraries/sendtohelix-browser.targets

```diff
@@ -233,17 +231,12 @@
     <ItemGroup Label="Add samples">
       <_WasmWorkItem Include="$(TestArchiveRoot)browseronly/**/*.zip"     Condition="'$(Scenario)' == 'WasmTestOnChrome'" />
```

Comment:
We don't need `browseronly`. Now we can make a do with `chromeonly` because it's triggered with `WasmTestOnChrome`. All the places with
`<TestArchiveTestsRoot>$(TestArchiveRoot)browseronly/</TestArchiveTestsRoot>`
need a change accordingly

Reply:
`browseronly` means FF too